recipe/meta.yaml

```
@@ -29,17 +29,22 @@ requirements:
     - wheel
   run:
     - python
-    - flask >=1.0.4
+    - flask >=1.0.4,<3.1
+    - werkzeug <3.1
     - plotly >=5.0.0
-    - importlib-metadata ==4.8.3  # [py<37]
-    - contextvars ==2.4  # [py<37]
+    - importlib-metadata
+    - typing_extensions >=4.1.1
+    - requests
+    - retrying
+    - ansi2html
+    - nest-asyncio
     # compress extra
     - flask-compress
   run_constrained:
     - dash-core-components ==2.0.0
     - dash-html-components ==2.0.0
     - dash-table ==5.0.0
```

More run
```suggestion
    - dash-table ==5.0.0
    - diskcache >=5.2.1
    - multiprocess >=0.70.12
    - psutil >=5.8.0
    - redis >=3.5.3
    - celery >=5.1.2
    - importlib-metadata <5 # [py<38]
```
https://github.com/plotly/dash/blob/v2.14.2/requires-diskcache.txt
https://github.com/plotly/dash/blob/v2.14.2/requires-celery.txt
https://github.com/plotly/dash/blob/v2.14.2/setup.py#L34-L35

---

The celery and diskcache requirements are optional dependencies for extras, but I could add them to run_constrained. `flask-compress` is also optional, but it's been included in past versions, so I left it in.